Libraries/Widget/ToolTip.hpp

```diff
@@ -29,8 +29,6 @@ struct ToolTipPlacement
   IndicatorSide::Type mPriority[4];
 };
 
-DeclareEnum6(ToolTipColorScheme, Default, Gray, Red, Yellow, Green, Orange);
```

Comment:
Does this really need moving?

Reply:
Due to Tooltip being a composite and composite requiring the enum yes.